.ci-scripts/test-for-licenseHeaders.sh

```diff
@@ -69,6 +69,7 @@ getFiles() {
 }
 
 while read file ; do
+  echo "$file"
```

Comment:
if the file fails it is written in the testFile() method, together with a reason. 
This addition is not really bringing any other functionality except for a list of all files tested.
Additonally, this is not relevant to the other changes and should therefore be a separate commit- if kept - I am open to a discussion

Reply:
I am sorry, this change was unintended. I will remove it.